specification/paloaltonetworks/resource-manager/PaloAltoNetworks.Cloudngfw/preview/2024-02-07-preview/PaloAltoNetworks.Cloudngfw.json

```diff
@@ -5035,21 +5033,21 @@
         "changes"
       ]
     },
-    "CloudManagerTenant": {
+    "CloudManagerTenantList": {
       "type": "object",
```

Comment:
Doesn't look like a dictionary to me rather just an object with a property called value which is an array. Is my understanding correct?

Reply:
It is very similar to what Storage RP returns for keys: StorageAccountListResult

Reply:
Yes, basically at the top level, RPaaS is expecting a name, value pair... and not a direct Array.
I was getting this error, from which I inferred that the array is the issue:

"Http request failed with unhandled exception of type 'JsonSerializationException' and message: 'Cannot deserialize the current JSON array (e.g. [1,2,3]) into type 'System.Collections.Generic.Dictionary`2[System.String,System.Object]' because the type requires a JSON object (e.g. {\"name\":\"value\"}) to deserialize correctly.\nTo fix this error either change the JSON to a JSON object (e.g. {\"name\":\"value\"}) or change the deserialized type to an array or a type that implements a collection interface (e.g. ICollection, IList) like List<T> that can be deserialized from a JSON array. JsonArrayAttribute can also be added to the type to force it to deserialize from a JSON array.\nPath ''.'."